docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -0,0 +1,707 @@
+
+````
+docker run -d -i -t  -p 8080:8080 lb4-simple-web-app
+````
+
+This starts the container in detached mode (-d), connects your machine's 8080 port to the 8080 port of the container (-p), and permits you to open a bash shell into the container later on if you'd like (-i -t).
+
+
+Afterwards, type:
+
+````
+docker ps
+````
+
+You should see something like this:
+
+````
+
+CONTAINER ID   IMAGE                COMMAND          CREATED             STATUS         PORTS
+a9962339e863   lb4-simple-web-app   "npm start"      8 seconds ago       Up 7 seconds   0.0.0.0:8080->8080/tcp
+
+````
+
+To see the log output of your container, you can type:
+
+````
+docker logs <container id>    e.g. a9962339e863
+````
+
+You should see something like:
+
+````
+> deploy-test@1.0.0 start /
+> node .
+
+Server is running at http://127.0.0.1:8080
+Try http://127.0.0.1:8080/ping
+````
+
+Open your browser and attempt to access all these urls
+
+http://127.0.0.1:8080/
+
+http://127.0.0.1:8080/ping
+
+http://127.0.0.1:8080/explorer
+
+http://127.0.0.1:8080/openapi.json
+
+
+## Step 10: Stop the application running in Docker
+
+Find the container id
+
+````
+docker ps | grep lb4
+````
+
+You should see something like this:
+
+````
+a9962339e863        lb4-simple-web-app      "npm start"
+````
+
+The leftmost value is the container id.
+
+Type:
+
+````
+docker stop <container id>  e.g. a9962339e863
+````
+
+## Step 11: Comment out the 'EXPOSE 8080' in the Dockerfile
+Before deploying this application to Kubernetes on the IBM Cloud, let's first comment out
+the 'EXPOSE 8080' line in the Dockerfile
+
+````
+#EXPOSE 8080
+````
+
+Build the docker image again:
+
+````
+docker build -t lb4-simple-web-app .
+````
+
+We will be uploading this image to the IBM Cloud Container Registry shortly.
+
+
+## Step 12: Log into IBM Cloud using ibmcloud login command
+
+Use ````ibmcloud login```` command to login.
+
+Side note: Since I’m using a federated user id (IBMid), I use
+````ibmcloud login --sso```` to login.
+
+After you’ve been successfully logged in, you’ll see something like:
+
+````
+API endpoint:      https://api.ng.bluemix.net
+Region:            us-south
+User:              dremond@ca.ibm.com
+Account:           Dominique Emond's Account
+Resource group:    default
+CF API endpoint:
+Org:
+Space:
+````
+
+## Step 13: Log into IBM Cloud Container Registry
+
+````
+ibmcloud cr login
+````
+
+You should see:
+
+````
+Logging in to 'registry.ng.bluemix.net'...
+Logged in to 'registry.ng.bluemix.net'.
+OK
+````
+
+## Step 14: Upload a docker image to the Container Registry
+
+This requires several steps, let's quickly go through them.
+
+### Create a namespace
+
+List your current namespace by typing:
+
+```
+ibmcloud cr namespace-list
+```
+
+If you want to create a new namespace for yourself, you can do so with this command:
+
+````
+ibmcloud cr namespace-add <my_namespace>
+````
+
+NOTE : I will be using my namespace ````dremond```` in the commands that follow.
```

Comment:
Please use "note" or "tip" alert here.

See https://loopback.io/doc/en/contrib/alerts.html#types-of-alerts

Reply:
done.